src/main/resources/org/jenkinsci/plugins/parallel_test_executor/SplitStep/help.html

```diff
@@ -1,4 +1,4 @@
 <div>
     Asks Jenkins to analyze the timing of tests from the last build and divide the tests for this build into roughly equal subsets for <code>parallel</code> execution.
-    <a href="https://github.com/jenkinsci/parallel-test-executor-plugin/blob/master/demo/repo/Jenkinsfile" target="_blank">example <code>Jenkinsfile</code></a>
+    <a href="https://github.com/jenkinsci/parallel-test-executor-plugin/blob/c57b597b8f032d6f9b0bd2fbb03cf067b116e56f/demo/repos/demo/Jenkinsfile" target="_blank">example <code>Jenkinsfile</code></a>
```

Comment:
Hmm, but then this just calls into a library https://github.com/jenkinsci/parallel-test-executor-plugin/blob/c57b597b8f032d6f9b0bd2fbb03cf067b116e56f/demo/lib/vars/testInParallel.groovy#L2-L29 and you sort of have to read the whole thing to get the context. Maybe it would be better to just keep this out of inline help
```suggestion
```